internal/controller/replication.storage/status.go

```diff
@@ -76,24 +83,28 @@ func setFailedPromotionCondition(conditions *[]metav1.Condition, observedGenerat
 // sets conditions when volume promotion was failed due to failed validation.
 func setFailedValidationCondition(conditions *[]metav1.Condition, observedGeneration int64) {
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "failed to promote",
 		Type:               v1alpha1.ConditionCompleted,
 		Reason:             v1alpha1.FailedToPromote,
 		ObservedGeneration: observedGeneration,
 		Status:             metav1.ConditionFalse,
 	})
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "error detected while promotion",
 		Type:               v1alpha1.ConditionDegraded,
 		Reason:             v1alpha1.Error,
 		ObservedGeneration: observedGeneration,
 		Status:             metav1.ConditionTrue,
 	})
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "Volume is not resyncing",
```

Comment:
This message is capitalised, while other are not. I'm not sure what is the wanted format but all messages must be consistent.

Reply:
ohh missed this, will update